docs/tutorial/visualization.rst

```diff
@@ -0,0 +1,420 @@
+*************
+Visualization
+*************
+
+This tutorial is about visualization of COMPAS objects in Rhino, Grasshopper, and Blender.
+For information about standalone 2D visualizations (i.e. "plotting"), please visit the tutorial about :ref:`tutorial_plotters`.
+For information about standalone 3D visualizations, please visit the documentation of :py:mod:`compas_view2`: https://compas.dev/compas_view2.
+
+
+Object - Artist Pairs
```

Comment:
The target audience of this doc is more like end-user, rather than developer, and while knowing the object/artist pairing is interesting, the title of the first item here should be something more squarely aimed at answering `What is an artist`.